php/src/Docker/DockerActionManager.php

```diff
@@ -499,6 +499,9 @@ public function CreateContainer(Container $container) : void {
                 }
                 $mounts[] = ["Type" => "bind", "Source" => $volume->name, "Target" => $volume->mountPoint, "ReadOnly" => !$volume->isWritable, "BindOptions" => [ "Propagation" => "rshared"]];
             }
+        // Special things for the watchtower and docker-socket-proxy container which should not be exposed in the containers.json
```

Comment:
can I ask why not using the conatiners.json?

Reply:
and why not doing that?

Reply:
because usually I am only adding stuff there that could be useful for further components like for https://github.com/nextcloud/all-in-one/discussions/1581. But I dont see this making sense for that...

Reply:
We can also still add it later